application/resources/views/auth/register.blade.php

```diff
@@ -43,24 +45,39 @@
                             <label for="password" class="col-md-4 col-form-label text-md-right">パスワード</label>
 
                             <div class="col-md-6">
-                                <input id="password" type="password" class="form-control @error('password') is-invalid @enderror" name="password" autocomplete="new-password">
+                                <input id="password" type="password"
+                                    class="form-control @error('password') is-invalid @enderror" name="password"
+                                    autocomplete="new-password">
 
                                 @error('password')
-                                    <span class="invalid-feedback" role="alert">
-                                        <strong>{{ $message }}</strong>
-                                    </span>
+                                <span class="invalid-feedback" role="alert">
+                                    <strong>{{ $message }}</strong>
+                                </span>
                                 @enderror
                             </div>
                         </div>
 
                         <div class="form-group row">
-                            <label for="password-confirm" class="col-md-4 col-form-label text-md-right">パスワード(確認)</label>
+                            <label for="password-confirm"
+                                class="col-md-4 col-form-label text-md-right">パスワード(確認)</label>
 
                             <div class="col-md-6">
-                                <input id="password-confirm" type="password" class="form-control" name="password_confirmation" autocomplete="new-password">
+                                <input id="password-confirm" type="password" class="form-control"
+                                    name="password_confirmation" autocomplete="new-password">
                             </div>
                         </div>
 
+                        {{-- イメージ登録 --}}
+                        <div class="form-group row">
+                            <label for="icon" class="col-md-4 col-form-label text-md-right">イメージ</label>
+
+                            <div class="col-md-6">
+                                <input id="icon" type="file" class=" " name="icon" value="{{ old('icon') }}"
+                                    autocomplete="icon" autofocus>
```

Comment:
```suggestion
                                <input id="icon" type="file" class=" " name="icon"
                                    autocomplete="icon" autofocus>
```
セキュリティの制約でinput.fileにvalueは指定できません。
どんなブラウザでも同じです。
指定しても無効化されます。

Reply:
レビューをしていただきありがとうございます。
大変勉強になりました。
修正いたします。